lib/tf_lock/tf_lock_acquire.py

```diff
@@ -43,12 +44,19 @@ def tf_lock_acquire(root_module: Path) -> ExitCode:
                 )
                 return 0
 
+            tf_lock_user = TFLockUser.from_string(lock_user)
+            # a pr holds the lock.
+            if tf_lock_user.pr_url:
```

Comment:
nit: Normally I put a comment about a particular line on that line, when I can:

```suggestion
            if tf_lock_user.pr_url:  # a pr holds the lock.
```